docs/getting-started.rst

```diff
@@ -18,10 +18,12 @@ Prerequisites
 Installation
 ------------
 
-To install ``xcdat`` using `conda`_, run:
+We recommend using the anaconda environment creation procedure to install ``xcdat``. The advantage of following this approach, is that any dependencies (e.g. ``python >= 3.8``) will be resolved during the environment creation. To proceed, run:
 
-.. code-block:: console
+To create an ``xcdat`` anaconda environment, run:
 
-    >>> conda install -c conda-forge xcdat
+.. code-block:: console
 
+    >>> conda create -n <xcdat> -c conda-forge xcdat
+    
 .. _conda: https://docs.conda.io/en/latest/
```

Comment:
If the above looks good to you, you can paste it over your edits.

Reply:
Also, the first option is technically a Conda environment with `xcdat` (and other user-specified packages installed), rather than an "xcdat anaconda environment".